docs/database-connection-string.md

```diff
@@ -1,85 +1,83 @@
+- Communication is set up with the `scdata` process.
 
-**Notes:**
+_For more information, see the article about [database processes](database-processes.md)._
 
-Database path is case sensitive even on the case insensitive operating systems (Windows).
-Attempt to establish multiple connections to the same database using different path casing, like `C:\Databases\TestDatabase` and `c:\databases\testdatabase`, will fail.
+**Notes:**
 
-Starcounter identifies database by the provided path, not by its physical location.
-Attempt to establish multiple connections to the same database using different paths will fail.
+The database path string is **case sensitive**, even on operating systems like Windows that use case insensitive file paths. Attempting to establish multiple connections to the same database using different path casing, like `C:\Databases\TestDatabase` and `c:\databases\testdatabase`, will fail.
 
-#### Windows
+Starcounter identifies database by the provided path, not by its physical location. Attempt to establish multiple connections to the same database using different paths will fail.
 
-It is not possible to delete database files while `scdata` process is running.
+### OpenMode
 
-#### Linux
+The OpenMode option defines how to interpret the database path on connection. It can have the following string values:
 
-It is possible to delete database files while `scdata` is running, which may lead to a trap.
+Value               | Description
+:------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`Open`              | Opens the database directory from the path specified in the Database option, if it exist. If the directory does not exist, or if there is no Starcounter database located there, an exception is thrown.
+`CreateIfNotExists` | Opens the database directory from the path specified in the Database option, if it exist. If the directory does not exist, or if there is no Starcounter database located there, it will create the necessary directories and database files.
 
-Consider the following scenario:
+### StartMode
 
-- A database is created at `/home/database/TestDatabase`.
-- An `scdata` process is started for the database.
-- The database files are deleted.
-- New database is created in exactly the same location.
-- An app tries to connect to the new database.
-- Starcounter detects, that there is a running `scdata` process associated with this database path.
-- The app connects to the existing `scdata` process, which still operates on the deleted files.
+The StartMode option defines how to start and/or connect to the database process once the database files have been located and/or created.
```

Comment:
```suggestion
The `StartMode` option defines how to start and/or connect to the database process once the database files have been located and/or created.
```

Reply:
OK, let's use syntax quotes for connection string options. I didn't want to over-use it, but sure :)

Reply:
(no need to add suggestions for it, I'll fix it)

Reply:
I prefer to use syntax quotes for code pieces to avoid unnecessary grammar checks. StartMode is not a correct word according to English grammar, but when it's `StartMode`, it's easier to filter out.

My personal preference.

Reply:
I don't think grammar checks is a good enough reason to use syntax quotes. It should be used for syntactic constructions that represent some formal notion in a system. I think these options, qualify, so I'll change it.